app/api/ada/lib/storage/adaLocalStorage.js

```diff
@@ -161,9 +161,11 @@ async function _saveInStorage(key: string, toSave: any): Promise<void> {
   await setLocalItem(key, JSON.stringify(toSave));
 }
 
-async function _getFromStorage(key: string): any | typeof undefined {
+async function _getFromStorage(key: string): any | void {
```

Comment:
I couldn’t figure out when this could be `void`

Reply:
`typeof undefined` ===  `void` in Flow

Reply:
@SebastienGllmt I’m not sure about the previous type either

Reply:
if you look for a key that doesn't exist, it returns `undefined`